MarsMission/Controllers/DetailsViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  DetailsViewController.swift
+//  MarsMission
+//
+//  Created by Thanyani on 2020/11/03.
+//
+
+import UIKit
+
+class DetailsViewController: UICollectionViewController {
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
```

Comment:
Please fix indentation and spacing

Reply:
<img width="1168" alt="Screenshot 2020-11-06 at 10 44 58" src="https://user-images.githubusercontent.com/55865505/98345568-8310f880-201d-11eb-9313-ae2c9c0fb83a.png">